code/game/jobs/job/civilian.dm

```diff
@@ -18,14 +18,11 @@
 		if(4) H.equip_or_collect(new /obj/item/weapon/storage/backpack/satchel(H), slot_back)
 	H.equip_or_collect(new /obj/item/clothing/under/color/random(H), slot_w_uniform)
 	H.equip_or_collect(new /obj/item/clothing/shoes/black(H), slot_shoes)
-	if(H.backbag == 1)
-		H.equip_or_collect(new /obj/item/weapon/storage/box/survival(H), slot_r_hand)
-	else
-		H.equip_or_collect(new /obj/item/weapon/storage/box/survival(H.back), slot_in_backpack)

```

Comment:
I've removed all instances of `H.back` where I could. It doesn't really matter where you spawn the item if you spawn it for the sole purpose of equipping it to a specific slot.

`equip_or_collect()` without any additional backpack checks does that more cleanly. What I did just makes it all more readable.